tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -1703,6 +1703,63 @@ public:
    }
 
+   ////////////////////////////////////////////////////////////////////////////
+   /// \brief Fill and return a GraphAsymmErrors (*lazy action*).
+   /// \param[in] v1Name The name of the column that will fill the x axis.
+   /// \param[in] v2Name The name of the column that will fill the y axis.
+   /// \param[in] eXLName The name of the column of X low errors
+   /// \param[in] eXHName The name of the column of X high errors
+   /// \param[in] eYLName The name of the column of Y low errors
+   /// \param[in] eYHName The name of the column of Y high errors
+   /// \return the GraphAsymmErrors wrapped in a RResultPtr.
+   ///
+   /// Columns can be of a container type (e.g. std::vector<double>), in which case the graph
+   /// is filled with each one of the elements of the container.
+   /// If Multithreading is enabled, the order in which points are inserted is undefined.
+   ///
+   /// This action is *lazy*: upon invocation of this method the calculation is
+   /// booked but not executed. Also see RResultPtr.
+   ///
+   /// ### Example usage:
+   /// ~~~{.cpp}
+   /// // Deduce column types (this invocation needs jitting internally)
+   /// auto myGAE1 = myDf.GraphAsymmErrors("xValues", "yValues", "exl", "exh", "eyl", "eyh");
+   /// // Explicit column types
+   /// using f = std::vector<float>;
+   /// auto myGAE2 = myDf.GraphAsymmErrors<f, f, f, f, f, f>("xValues", "yValues", "exl", "exh", "eyl", "eyh");
+   /// ~~~
+   ///
+   /// \note Differently from other ROOT interfaces, the returned graph is not associated to gDirectory
+   /// and the caller is responsible for its lifetime (in particular, a typical source of confusion is that
+   /// if result histograms go out of scope before the end of the program, ROOT might display a blank canvas).
+   template <typename V1 = RDFDetail::RInferredType, typename V2 = RDFDetail::RInferredType,
+             typename EXL = RDFDetail::RInferredType, typename EXH = RDFDetail::RInferredType,
+             typename EYL = RDFDetail::RInferredType, typename EYH = RDFDetail::RInferredType>
+   RResultPtr<::TGraphAsymmErrors>
+   GraphAsymmErrors(std::string_view v1Name = "", std::string_view v2Name = "", std::string_view eXLName = "",
+                    std::string_view eXHName = "", std::string_view eYLName = "", std::string_view eYHName = "")
+   {
+      auto graph = std::make_shared<::TGraphAsymmErrors>();
+      const std::vector<std::string_view> columnViews = {v1Name, v2Name, eXLName, eXHName, eYLName, eYHName};
+      const auto userColumns = RDFInternal::AtLeastOneEmptyString(columnViews)
+                                  ? ColumnNames_t()
+                                  : ColumnNames_t(columnViews.begin(), columnViews.end());
+
+      const auto validatedColumns = GetValidatedColumnNames(6, userColumns);
+
+      // We build a default name and title based on the input columns
+      if (!(validatedColumns[0].empty() && validatedColumns[1].empty())) {
```

Comment:
I don't think this can every be true? otherwise what columns would we be filling the graph with?

Reply:
Hmm, maybe I agree, here I was thinking:  Can we fail with the assumption that `userColumns` are always the same as `validatedColumns`? If they are always the same, the code could be simplified.